vllm/model_executor/layers/quantization/utils/mxfp4_utils.py

```diff
@@ -99,6 +100,14 @@ def _can_support_mxfp4(
     )
 
 
+def get_padding_alignment():
+    return (
+        256
+        if triton.runtime.driver.active.get_current_target().arch in ("gfx950",)
```

Comment:
The default value was 256. Will it be safer to only update MI300's alignment to 128? Or do you think 128 will be faster on other architectures?